openmc/deplete/microxs.py

```diff
@@ -91,7 +92,11 @@ def from_model(cls,
         model.tallies = tallies
 
         # create temporary run
-        with tempfile.TemporaryDirectory() as temp_dir:
+        if 'TMPDIR' in os.environ:
+            d = tempfile.gettempdir()
+        else:
+            d = Path.cwd()
+        with tempfile.TemporaryDirectory(dir=d) as temp_dir:
```

Comment:
I'm not sure we want to go down this rabbit hole. Python actually lets you use one of three different environment variables: `TMPDIR`, `TEMP`, or `TMP`. Rather than making more changes here, I think we should either 1) stick with the original one-line change or 2) close this PR and simply instruct users to specify `TMPDIR` if they are on a cluster where there may be issues with local filesystems not being shared across nodes (perhaps replace with a PR that updates our documentation with such a recommendation).